src/Nethermind/Nethermind.Evm.Test/Eip2935Tests.cs

```diff
@@ -28,7 +28,7 @@ public override void Setup()
 
 
     [TestCase(MainnetSpecProvider.CancunBlockTimestamp, false)]
-    [TestCase(MainnetSpecProvider.PragueBlockTimestamp, true)]
+    [TestCase(MainnetSpecProvider.PragueBlockTimestamp, false)]
     public void CorrectBlockhashBeingUsed(ulong timestamp, bool eipEnabled)
```

Comment:
here eipEnabled=false does not make sense
eip2935 is enabled - just the expectation changed
i think this test can be removed or modify it for using 7709